docs/self-managed/backup-restore/operate-tasklist-backup.md

```diff
@@ -32,18 +30,29 @@ Before you can use the backup and restore feature:
 
 ```yaml
 for Operate:
-camunda.operate.backup.repositoryName=<repository name>
+camunda.operate: backup.repositoryName=<repository name>
```

Comment:
Shouldn't it be `camunda.operate.backup.repositoryName=<repository name>` in yaml, or did you mean this?
```yaml
camunda.operate:
  backup:
    repositoryName: <repository name>
```

Reply:
Thank you, Ralf!
I think I lost the line breaks. I meant
```
camunda.operate:
  backup.repositoryName: <repository name>
```
But now thinking of merging is back as before.